packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/multiplier.tsx

```diff
@@ -7,13 +7,12 @@ const Multiplier = observer(() => {
     const { multiplier, multiplier_range_list, onChange } = useTraderStore();
     return (
         <Dropdown
-            id='multiplier'
+            test_id='multiplier'
             className='trade-container__multiplier-dropdown'
             is_alignment_left
-            is_nativepicker={false}
-            list={multiplier_range_list}
+            list={multiplier_range_list as unknown as React.ComponentProps<typeof Dropdown>['list']}
```

Comment:
I'm using assertion here because Dropdown is currently typed in a way that it only accepts lists with string values only but in fact, it works well with numeric values.